apps/coa/views/content/en/what-counts-as-proof-of-identity.md

```diff
@@ -0,0 +1,17 @@
+<details class="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      What counts as proof of identity?
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+    <ul>
```

Comment:
`Acceptable proof of identity documents include: `
The above line has to be added.